docs/index.md

```diff
@@ -1,5 +1,17 @@
-# PowSyBl's subprojects  
+# PowSyBl's readthedocs subprojects  
+
+- [powsybl-core](https://powsybl.readthedocs.io/projects/powsybl-core)
+- [powsybl-open-loadflow](https://powsybl.readthedocs.io/projects/powsybl-open-loadflow)
+- [powsybl-diagram](https://powsybl.readthedocs.io/projects/powsybl-diagram)
+- [powsysbl-open-rao](https://powsybl.readthedocs.io/projects/openrao)
+- [powsybl-dynawo](https://powsybl.readthedocs.io/projects/powsybl-dynawo)
+- [powsybl-entsoe](https://powsybl.readthedocs.io/projects/entsoe)
+
+- [pypowsybl](https://powsybl.readthedocs.io/projects/pypowsybl)
+- [pypowsybl-jupyter](https://powsybl.readthedocs.io/projects/pypowsybl-jupyter)
+
+- [powsybl-afs](https://powsybl.readthedocs.io/projects/powsybl-afs)
```

Comment:
Should be in a way with java repositories.

Reply:
As discussed, powsybl-afs is included in "advanced features" for now!